recirq/qml_lfe/learn_states_c_test.py

```diff
@@ -0,0 +1,45 @@
+import os
+import pytest
+import tempfile
+import numpy as np
+from . import learn_states_c
+
+
+def _predict_exp(data, pauli_list, two_copy):
```

Comment:
Might be worth a comment.  Also, is this the same as _predict_exp from the other tests?

Reply:
Added. It's the same.